FeedStoreChallenge/Core Data/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by YH Kung on 2021/7/8.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal final class ManagedFeedImage: NSManagedObject {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`. Please remove it from this class and its extension.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```